### PR TITLE
iOS implementation

### PR DESCRIPTION
**Pull request by jeremie-movify**
Bound all of Qualtrics' SDK methods to plugin

### CODE REVIEW

File: example/ios/Flutter/Generated.xcconfig [hunk 1]
```diff
@@ -1,9 +1,10 @@
 // This is a generated file; do not edit or check into version control.
```
Comment (timrijckaert) [inline example/ios/Flutter/Generated.xcconfig]: 👀

File: example/ios/Flutter/flutter_export_environment.sh [hunk 2]
```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
```
Comment (timrijckaert) [inline example/ios/Flutter/flutter_export_environment.sh]: 👀

File: example/ios/Flutter/Generated.xcconfig (hunk 1 above)
Comment (jeremie-movify) [reply example/ios/Flutter/Generated.xcconfig]: Yeah, pod install was failing if this file didn't exist 😭 I'm looking into it

**Review by timrijckaert — APPROVED** [review]

File: example/ios/Flutter/Generated.xcconfig (hunk 1 above)
Comment (jeremie-movify) [reply example/ios/Flutter/Generated.xcconfig]: ✅

File: example/ios/Flutter/flutter_export_environment.sh (hunk 2 above)
Comment (jeremie-movify) [reply example/ios/Flutter/flutter_export_environment.sh]: ✅

**Review by petermnt — APPROVED** [review]

File: ios/Classes/SwiftQualtricsFlutterPlugin.swift [hunk 3]
```diff
@@ -1,14 +1,60 @@
...
+          }
+        }
+      case .display:
+        let rootViewController = UIApplication.shared.keyWindow!.rootViewController!
+        let displayResult = Qualtrics.shared.display(viewController: rootViewController)
+        result(displayResult)
+      case .displayTarget(let arguments):
+        let rootViewController = UIApplication.shared.keyWindow!.rootViewController!
+        Qualtrics.shared.displayTarget(targetViewController: rootViewController, targetUrl: arguments.target)
+        result(true)
```
Comment (Kymer) [inline ios/Classes/SwiftQualtricsFlutterPlugin.swift:42]: Does `Qualtrics.shared.displayTarget` not return a result as `Qualtrics.shared.display` does?

File: ios/Classes/SwiftQualtricsFlutterPlugin.swift [hunk 4]
```diff
@@ -1,14 +1,60 @@
...
+          if let targetingResultError = targetingResult.getError() {
+            result(targetingResultError.asFlutterError)
+          }
+          else {
+            result(targetingResult.passed())
+          }
+        }
+      case .display:
+        let rootViewController = UIApplication.shared.keyWindow!.rootViewController!
+        let displayResult = Qualtrics.shared.display(viewController: rootViewController)
```
Comment (Kymer) [inline ios/Classes/SwiftQualtricsFlutterPlugin.swift:37]: What does the displayResult of `Qualtrics.shared.display` signify?

**Review by Kymer — APPROVED** [review]

File: ios/Classes/SwiftQualtricsFlutterPlugin.swift (hunk 4 above)
Comment (jeremie-movify) [reply ios/Classes/SwiftQualtricsFlutterPlugin.swift:37]: I couldn't find any documentation on the return value ; assuming its meaning is whether the survey was effectivevely displayed 🤷‍♂️

File: ios/Classes/SwiftQualtricsFlutterPlugin.swift (hunk 3 above)
Comment (jeremie-movify) [reply ios/Classes/SwiftQualtricsFlutterPlugin.swift:42]: Actually no :
`@objc public func display(viewController: UIViewController) -> Bool`
`@objc public func displayTarget(targetViewController: UIViewController, targetUrl: String)`

**Pull request merged**